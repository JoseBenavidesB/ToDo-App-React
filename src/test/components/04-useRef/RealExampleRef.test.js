import React from 'react'
import { shallow } from 'enzyme'
import { RealExampleRef } from '../../../components/04-useRef/RealExampleRef';




describe( ' test RealExampleRef ', () => {

    const wrapper = shallow( <RealExampleRef /> );

    test('should return correctly ', () => { 
        
        expect( wrapper ).toMatchSnapshot();
        expect( wrapper.find('MultipleCustomHooks').exists() ).toBe(false);
    });

    test('should show <MultipleCustomHook /> ', () => { 
        
        wrapper.find('button').simulate('click')
        expect( wrapper.find('MultipleCustomHooks').exists() ).toBe(true);

        console.log(wrapper.html())

    });
})