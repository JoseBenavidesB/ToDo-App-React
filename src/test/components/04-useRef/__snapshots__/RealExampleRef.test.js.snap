// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` test RealExampleRef  should return correctly  1`] = `
<Fragment>
  <h1>
     Real Example Ref
  </h1>
   
  <hr />
  <button
    className="btn btn-danger"
    onClick={[Function]}
  >
    Show/Hide
  </button>
</Fragment>
`;
